fec/management/commands/load_bulk.py

```diff
@@ -28,15 +29,25 @@ def handle(self, *args, **options):
                 bulkfile, headers, delimiter='|', quotechar='"')
 
             i = 0
-
             for row in reader:
                 i += 1
-                record_creator(row)
-                print('.', end='', flush=True)
-
-            print("loaded %d '%s' records" % (i, fec_type))
-        # except Exception as exception:
-        #     self.stderr.write(str(exception))
+                try:
+                    record_creator(row)
+                    print(
+                        "\rloaded %d '%s' records" % (i, fec_type),
+                        flush=True,
+                        end='')
+                except Exception as exception:
+                    bad_rows_path = '{}_bad_rows.txt'.format(
+                        os.path.splitext(filename)[0])
```

Comment:
nice, i like that the error file will have the a similar filename to the one you're importing